jest.test.config.js

```diff
@@ -1,5 +1,42 @@
+// Resolve the absolute path of the caller location.
+const rootPath = process.cwd();
+
 module.exports = {
-  preset: '@commercetools-frontend/jest-preset-mc-app',
+  displayName: 'test',
+  globals: {
+    'process.env': {
+      NODE_ENV: 'test',
+    },
+  },
   moduleDirectories: ['src', 'node_modules'],
+  moduleNameMapper: {
+    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
+      '<rootDir>/test/transform-file.js',
+    '\\.css$': 'identity-obj-proxy',
+  },
+  rootDir: rootPath,
+  setupFiles: [
+    'raf/polyfill',
+    '<rootDir>/test/setup-tests.js',
+    'jest-localstorage-mock',
+  ],
   setupTestFrameworkScriptFile: '<rootDir>/scripts/setup-test-framework.js',
+  snapshotSerializers: ['enzyme-to-json/serializer'],
+  testEnvironment: 'jsdom',
+  testURL: 'https://mc.commercetools.com/',
+  testPathIgnorePatterns: ['node_modules', 'cypress'],
+  testRegex: '\\.spec\\.js$',
+  transform: {
+    '^.+\\.js$': '<rootDir>/test/transform-babel-jest.js',
+  },
+  transformIgnorePatterns: [
+    // This option tells Jest to ignore specific folders from being transpiled
+    // (e.g. with babel).
+    // However we need to instruct jest to actually transpile some packages.
+    // NOTE: this might not be necessary anymore once we ship packages already
+    // transpiled.
+    'node_modules/(?!(@commercetools-frontend)/)',
```

Comment:
not sure we need this?

Reply:
We are not using any `@commercetools-frontend` packages anymore, so we can remove this 👍